library/coverage/result_spec.rb

```diff
@@ -0,0 +1,34 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe 'Coverage.result' do
+  before :all do
+    @config_file = File.expand_path('../fixtures/start_coverage.rb', __FILE__)
```

Comment:
There is an actual `fixture` helper which should do exactly the same :wink: , as in https://github.com/ruby/spec/blob/master/core/module/autoload_spec.rb#L18.